extensions/amp-iframe/0.1/test/test-amp-iframe.js

```diff
@@ -235,7 +236,8 @@ describes.realWin(
         expect(iframe.getAttribute('referrerpolicy')).to.equal('no-referrer');
         expect(iframe.getAttribute('frameborder')).to.equal('3');
         expect(iframe.getAttribute('tabindex')).to.equal('-1');
-        // unsupproted attributes
+        expect(iframe.getAttribute('title')).to.equal('example title');
```

Comment:
Can we also add a test for when the `title` is mutated since this change is propagating it?